src/main/java/uk/gov/hmcts/reform/coh/service/QuestionRoundService.java

```diff
@@ -117,14 +122,17 @@ protected boolean isMaxRoundLimit(int maxQuestionRounds) {
     @Transactional
     public QuestionRoundState retrieveQuestionRoundState(QuestionRound questionRound) {
         List<Question> questions = questionRound.getQuestionList();
-        if(questions.isEmpty()) {
+        if (questions.isEmpty()) {
             Optional<QuestionState> optionalDraftedState = questionStateService.retrieveQuestionStateByStateName(DRAFTED);
-            if(!optionalDraftedState.isPresent()) {
+            if (!optionalDraftedState.isPresent()) {
                 throw new NoSuchElementException("Error: Required state not found.");
             }
             return new QuestionRoundState(optionalDraftedState.get());
         }
 
```

Comment:
Still to try out the LazyCollection workaround for the multi-bag problem. 
This change is to grab the answers for questions for both types of QR GET.